controllers/alias.go

```diff
@@ -64,6 +65,9 @@ type MachineReconciler struct {
 
 	// WatchFilterValue is the label value used to filter events prior to reconciliation.
 	WatchFilterValue string
+
+	// DrainKubeTimeout is the maximum length of time to wait for a response before giving up on kubernetes to drain a node.
+	DrainKubeTimeout time.Duration
```

Comment:
```suggestion
	NodeDrainClientTimeout time.Duration
```
(and in other places as well, I would also modify the godoc a bit similar to the flag description)